tools/lkl/cptofs.c

```diff
@@ -480,6 +480,19 @@ static int searchdir(const char *src, const char *dst, const char *match)
 	return ret;
 }
 
+static int match_root(const char *src)
+{
+	const char *c = src;
+
+	while (*c) {
+		if (*c != '.' && *c != '/')
```

Comment:
Just FYI, this will match "../" and "/../../../....." and other nonsense like that too. That's why I phrased my original comment as a question, FWIW: not sure it's actually worth doing this accurately 😄